Mage.Tests/src/test/java/org/mage/test/cards/single/c18/SowerOfDiscordTest.java

```diff
@@ -0,0 +1,141 @@
+package org.mage.test.cards.single.c18;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.multiplayer.MultiplayerTriggerTest;
+
+/**
+ * @author Susucr
+ */
+public class SowerOfDiscordTest extends MultiplayerTriggerTest {
+
+    /**
+     * {@link mage.cards.s.SowerOfDiscord Sower of Discord} {4}{B}{B}
+     * Creature — Demon
+     * Flying
+     * As Sower of Discord enters the battlefield, choose two players.
+     * Whenever damage is dealt to one of the chosen players, the other chosen player also loses that much life.
+     * 6/6
+     */
+    private static final String sower = "Sower of Discord";
+
+    @Test
+    public void test_Trigger_Bolt_First() {
+        setStrictChooseMode(true);
+
+        addCard(Zone.HAND, playerA, sower);
+        addCard(Zone.HAND, playerA, "Lightning Bolt");
+        addCard(Zone.BATTLEFIELD, playerA, "Badlands", 7);
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, sower, true);
+        addTarget(playerA, playerB, playerD);
```

Comment:
@JayDi85 tell me how to do those tests without the extra support for `^` on targetting player.

This is the target for a trigger, and it needs to target multiple players.